src/renderer/pages/flash/steps/firmware/CloudFirmwareReleasesPicker.tsx

```diff
@@ -9,26 +10,49 @@ type Props = {
   version?: string;
 };
 
-const GET_SIMPLE = gql(/* GraphQL */ `
-  query Simple($name: String) {
-    simple(name: $name) {
-      bloup
+const GET_FIRMWARES = gql(/* GraphQL */ `
+  query Firmwares {
+    cloudFirmwares {
+      releases {
```

Comment:
You don't need to redefine this in the cloud builds section. You can just write a query here which also queries the existing releases resolver.